index.html

```diff
@@ -2,38 +2,185 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <title>Portfolio</title>
+    <title>Mobile_and_desktop_version_portfolio</title>
+    <link rel="stylesheet" href ="styles.css">
 </head>
 <body>
     <header>
-        <div class="uni">
-            <a href="#">My Logo</a>
-            <img src="asset/Union.png">
-        </div>
+        <div class="header">
+        <a class="logo" href="#">My Logo</a> <img class="img" src="asset/Union.png" alt="Union">
+    </div>
     </header>
     <section class="intro">
-        <div class="greeting">
```

Comment:
Fantastic job on your home page and incorporating the entire project! However, let's concentrate on this first project for now. I have a few suggestions that could make it look even more impressive and aligned with the provided template:

- [ ] Would you mind checking the top space? It seems a tad larger than the template.🤓
- [ ] Could you please review the left padding for your text? You might consider reducing it slightly. 😄
- [ ] It would be great if you could align your social media icons. Currently, they appear a bit misaligned. 👍

   ### Your design:
   ![image](https://github.com/Surafels/Portfolio_Mobile/assets/107952724/0fe50a0b-1e99-4007-8f87-c6b377ec1df6)

   ### Figma design:
   ![image](https://github.com/Surafels/Portfolio_Mobile/assets/107952724/fec5ec57-494d-4577-ae03-c783aa531f7d)